postman/MIDAZ.postman_collection.json

```diff
@@ -2549,7 +2865,14 @@
 					],
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```

Comment:
what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.